emmo/ontology.py

```diff
@@ -69,7 +69,7 @@ def get_ontology(self, base_iri='emmo-inferred'):
             of EMMO
         """
         if base_iri == 'emmo':
-            base_iri = 'http://emmo.info/emmo'
+            base_iri = 'https://emmo-repo.github.io/latest-stable/emmo.owl'
         elif base_iri == 'emmo-inferred':
             base_iri = (
                 'https://emmo-repo.github.io/latest-stable/emmo-inferred.ttl')
```

Comment:
Probably a good idea to load the owl file instead of ttl to save temporary conversion and storage of the ontology.

Reply:
OK, done. Have to change this if emmo eventually stops supporting owl